src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -58,17 +58,17 @@
     These MonoDevelop dependencies don't ship with Visual Studio, so can't break our
```

Comment:
Comment still necessary now that it's codified in code? (No opinion either way on this one, just asking.)

Reply:
I think it's helpful to understand how the pattern is being used.